server/redisstate.go

```diff
@@ -194,8 +194,20 @@ func (s *RedisState) GetSenderOfTxHash(txHash string) (txSender string, found bo
 func (s *RedisState) AddTxToWhitehatBundle(bundleId string, signedTx string) error {
 	key := RedisKeyWhitehatBundleTransactions(bundleId)
 
+	// Check if item already exists
+	txs, err := s.GetWhitehatBundleTx(bundleId)
+	if err != nil {
+		return err
```

Comment:
do we want to fail here ? Because if there are some reason duplicate check failed and the tx is not duplicate,we might loose storing the valid tx. what do you think ?

Reply:
yeah good point!